packages/backend/src/apps/formsg/auth/decrypt-form-response.ts

```diff
@@ -69,6 +69,19 @@ export async function decryptFormResponse(
     return false
   }
 
+  // Note: this could occur due to pipe transfer since connection becomes null
+  if (!$.auth.data) {
+    logger.error(
+      'Form is not connected to any pipe after pipe is transferred',
+      {
```

Comment:
Can add an event so that it's easier to search for in DD
```suggestion
      {
        event: 'formsg-missing-private-key',
```

Reply:
ok boss i was thinking if need an event here